Tests/GRPCTests/GRPCReflectionServiceTests/Utils.swift

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2023, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import GRPC
+import GRPCReflectionService
+import NIOPosix
+import SwiftProtobuf
+
+public func generateProto(name: String, id: Int) -> Google_Protobuf_FileDescriptorProto {
```

Comment:
None of these functions need to be public

Reply:
This name doesn't tell me anything about the proto it generates nor what the name/id are for